rotkehlchen/tests/api/test_history.py

```diff
@@ -337,9 +344,19 @@ def test_query_pnl_report_events_pagination_filtering(
             assert x['timestamp'] >= events[idx + 1]['timestamp']
 
 
-@pytest.mark.parametrize('ethereum_accounts', [])
-def test_history_debug_export(rotkehlchen_api_server):
+@pytest.mark.parametrize('ethereum_accounts', [[]])
+@pytest.mark.parametrize('have_decoders', [[True]])
+def test_history_debug_export(rotkehlchen_api_server: 'APIServer') -> None:
     """Check that the format of the data exported matches the expected type."""
+    tx_id = '10' + str(make_evm_tx_hash())
```

Comment:
Why are you ignoring a random optimism transaction? Perhaps add a comment next to this that you want to check it later?

Reply:
The set that couldn't be json serialised came from the ignored events. What I did was create a event, add it to the list of ignored ones so when querying the endpoint the key is populated with such set